projects/js-packages/publicize-components/src/components/connection/index.js

```diff
@@ -47,7 +46,7 @@ class PublicizeConnection extends Component {
 	 *
 	 * @returns {boolean} True if connection must be reauthenticated.
 	 */
-	connectionNeedsReauth = () => includes( this.props.mustReauthConnections, this.props.name );
+	connectionNeedsReauth = () => this.props.mustReauthConnections.includes( this.props.name );
```

Comment:
I'm guessing not, but could `mustReauthConnections` ever be null?

Reply:
No, it will always be an array.